docs/MockserverAPI.md

```diff
@@ -18,13 +18,62 @@ You just have to return an array of objects that will be used as the initial dat
 - `getInitialDataSet?: (contextId: string) => object[];`
   - `contextId` represents the tenant id for the current request
 
+### fetchEntries
+
+This method allow to provide the GET request for entity set. It should be in the file named *<entitySetName>.js* where <entitySetName> is the name of the entity set.
+
+If you want to return a specifi file as result of your GET request, here is a sample code:
+```javascript
+module.exports = {
+  fetchEntries: function (keyValues, odataRequest) {
+    const fs = require('fs');
+    if (odataRequest.queryPath[0].path == '<entitySetName>') {
+      return JSON.parse(fs.readFileSync(process.cwd() + '\\webapp\\localService\\data\\Expand.json', 'utf8'));
+    }
+  },
+}
+```
+
+### addEntry
+
+This method allow to provide POST request (e.g. create entity) for entity set. It should be in the file named *<entitySetName>.js* where <entitySetName> is the name of the entity set.
+
+If you want to modify the request content of the create request:
+```javascript
+module.exports = {
```

Comment:
overriding the addEntry can be useful in some case yes but i would not keep the example you have there, the __metadata and uri are computed automatically and should not be maintained by hand.

Furthermore could you please align the documentation style with the other function ?  (including the signature and without the need for indicating that this should be in entitySetName.js since this is anyway covered globally (or should be)